de.metas.document.archive/de.metas.document.archive.base/src/main/java/de/metas/document/archive/api/impl/PlainDocOutboundDAO.java

```diff
@@ -67,4 +68,29 @@ public boolean accept(final I_C_Doc_Outbound_Log pojo)
 			}
 		});
 	}
+	
```

Comment:
with the "modern" query in place in AbstractDocOutboundDAO (see above), you can remove this method from PlainDocOutBoundDAO